kernel/packages/entryPoints/website.ts

```diff
@@ -46,12 +46,14 @@ import { startRealmsReportToRenderer } from 'unity-interface/realmsForRenderer'
 const logger = createLogger('website.ts: ')
 
 function configureTaskbarDependentHUD(i: UnityInterface, voiceChatEnabled: boolean) {
+  let worldConfig: WorldConfig = globalThis.globalStore.getState().meta.config.world!
```

Comment:
this is assuming that `globalThis.globalStore.getState().meta.config.world` will never be undefined, but after doing a quick look at the code, it might be undefined.
Maybe doing something this is better?

```suggestion
  const worldConfig: WorldConfig = globalThis.globalStore.getState().meta.config.world ?? {}
```

Reply:
I think the "??" operation doesn't work in Typescript

Reply:
😒 of course it does :P

Reply:
Done! take a look ;) Thanks for the feedback!